modules.yml

```diff
@@ -85,6 +85,10 @@ categories:
         slug: 'micronaut-azure'
         title: 'Micronaut Azure'
         description: 'Integration between Micronaut and Microsoft Azure'
+      object-storage:
+        slug: 'micronaut-object-storage'
+        title: 'Micronaut Object Storage'
+        description: 'Integration to read, update and delete objects. Supports Amazon S3, Azure Blog Storage, Google Cloud Storage, and Oracle Cloud Infrastructure Object Storage)'
```

Comment:
```suggestion
        description: 'Provides a uniform API to create, read and delete objects in the major cloud providers. Supports Amazon S3, Azure Blob Storage, Google Cloud Storage, and Oracle Cloud Infrastructure (OCI) Object Storage)'
```

Reply:
I had removed the `in the major cloud providers` to keep it short. similar to other descriptions